### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by AliabbasMerchant**
@YourTechBud Done

### CODE REVIEW

File: docs/manual/database/create.md [hunk 1]
```diff
@@ -156,75 +154,26 @@ HashMap[] docs = new HashMap[2];
 docs[0] = document;
 docs[1] = document2;
 db.insert("books").docs(docs).apply(new Utils.ResponseListener() {
-    @Override
```
Comment (YourTechBud) [inline docs/manual/database/create.md]: Why did you remove this?

Comment (YourTechBud) [reply docs/manual/database/create.md]: Seems like an error. Instead pass a handler sort of a thing to the apply function

Comment (YourTechBud) [reply docs/manual/database/create.md]: Do this in all similar occurrences

Comment (AliabbasMerchant) [reply docs/manual/database/create.md]: How is this?:  
`.apply(myResponseListener);`

Comment (YourTechBud) [reply docs/manual/database/create.md]: Perfect. We should do that

Comment (AliabbasMerchant) [reply docs/manual/database/create.md]: Cool
